projects/1.16/assets/integrated-tunnels/integratedtunnels/lang/zh_cn.json

```
@@ -1 +1,543 @@
+    "aspect.aspecttypes.integratedtunnels.boolean.fluid.nbtrecursive": "递推检查 NBT",
+    "aspect.aspecttypes.integratedtunnels.boolean.fluid.nbtrecursive.info": "嵌套 NBT 标签及列表是否应被递推检查",
+    "aspect.aspecttypes.integratedtunnels.boolean.energy.checkamount": "检测能量储量",
+    "aspect.aspecttypes.integratedtunnels.boolean.energy.checkamount.info": "只允许指定的能量储量在一个游戏刻中传输，而不是更少。",
+    "aspect.aspecttypes.integratedtunnels.boolean.world.blockupdate": "引起方块更新",
+    "aspect.aspecttypes.integratedtunnels.boolean.world.righthand": "右手",
+    "aspect.aspecttypes.integratedtunnels.boolean.world.silktouch": "精准采集",
+    "aspect.aspecttypes.integratedtunnels.boolean.world.ignorereplacable": "忽略可否替换",
+    "aspect.aspecttypes.integratedtunnels.boolean.world.breaknodrops": "当没有掉落物时破坏方块",
+    "aspect.aspecttypes.integratedtunnels.boolean.world.breaknodrops.info": "方块只有在产生掉落物的情况下才能被破坏",
+    "aspect.aspecttypes.integratedtunnels.boolean.world.ignorepickupdelay": "忽略拾取延迟",
+    "aspect.aspecttypes.integratedtunnels.boolean.world.dispense": "分配",
+    "aspect.aspecttypes.integratedtunnels.double.world.offsetx": "偏移X",
+    "aspect.aspecttypes.integratedtunnels.double.world.offsetx.info": "值在0和1之间。",
+    "aspect.aspecttypes.integratedtunnels.double.world.offsety": "偏移Y",
+    "aspect.aspecttypes.integratedtunnels.double.world.offsety.info": "值在0和1之间。",
+    "aspect.aspecttypes.integratedtunnels.double.world.offsetz": "偏移Z",
+    "aspect.aspecttypes.integratedtunnels.double.world.offsetz.info": "值在0和1之间。",
+    "aspect.aspecttypes.integratedtunnels.boolean.world.lifespan": "存留时间（刻）",
+    "aspect.aspecttypes.integratedtunnels.boolean.world.delaybeforepickup": "拾取延迟（刻）",
+    "aspect.aspecttypes.integratedtunnels.double.world.velocity": "速度",
+    "aspect.aspecttypes.integratedtunnels.double.world.velocity.info": "实体速度值在0到25之间。",
+    "aspect.aspecttypes.integratedtunnels.double.world.yaw": "偏转角",
+    "aspect.aspecttypes.integratedtunnels.double.world.yaw.info": "在-180到180度之间左/右旋转。",
+    "aspect.aspecttypes.integratedtunnels.double.world.pitch": "俯仰角",
+    "aspect.aspecttypes.integratedtunnels.double.world.pitch.info": "在-180到180度之间上/下旋转。",
+    "aspect.aspecttypes.integratedtunnels.boolean.player.rightclick": "右击",
+    "aspect.aspecttypes.integratedtunnels.integer.entityindex": "实体索引",
+    "aspect.aspecttypes.integratedtunnels.integer.entityindex.info": "将选择第n个实体。-1为随机实体。",
+    "aspect.aspecttypes.integratedtunnels.boolean.player.continuousclick": "连续点击",
+    "aspect.aspecttypes.integratedtunnels.boolean.player.continuousclick.info": "要么连续点击要么单次点击。",
+    "aspect.aspecttypes.integratedtunnels.boolean.blacklist": "黑名单",
+    "aspect.aspecttypes.integratedtunnels.boolean.blacklist.info": "处理所有指定列表外的物品。",
+    "aspect.aspecttypes.integratedtunnels.boolean.player.sneak": "潜行",
+    "aspect.aspecttypes.integratedtunnels.boolean.player.sneak.info": "是否模拟潜行中（按住 shift ）的玩家。",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
    "aspect.aspecttypes.integratedtunnels.boolean.player.sneak.info": "是否模拟潜行中（按住 shift）的玩家。",
```
此处无需空格。